_template_python/INDICATOR_DEV_GUIDE.md

```diff
@@ -324,6 +321,52 @@ Using this tag dictionary, we can interpret the following signals as
 * `confirmed_admissions_influenza_1d_prop` = raw (unsmoothed) daily ("1d") confirmed influenza hospital admissions ("confirmed_admissions_influenza") per 100,000 population ("prop").
 * `confirmed_admissions_influenza_1d_prop_7dav` = the same as above, but smoothed with a 7-day moving average ("7dav").
 
+#### Implement a Missing Value code system
+
+The column is described [here](https://cmu-delphi.github.io/delphi-epidata/api/missing_codes.html).
+
```

Comment:
Before discussing patching, I think it would be helpful to have a small section talking about what an issue is and how it's different from time value, mainly for people who aren't used to working with time series data.